### ISSUE TITLE
UN registers 'Türkiye' as new country name to replace 'Turkey'

### ISSUE THREAD

**Issue by alanorth** [pycountry/pycountry#148]
In 2022-06 [the UN accepted a request by the Turkish Foreign Minister to change the name of the country from "Turkey" to "Türkiye"](https://www.aa.com.tr/en/americas/un-registers-turkiye-as-new-country-name-to-replace-turkey/2603492). This has been updated in the [UN M.49](https://unstats.un.org/unsd/methodology/m49/) as well as the [Debian iso-codes package as of version 4.12.0](https://salsa.debian.org/iso-codes-team/iso-codes/-/blob/main/CHANGELOG.md#4120-2022-11-06).

Can you update it here too? Thank you!

**Comment by jkdaza** [comment]
We are also keen to see this changed, also.
Our client is using [Indico](https://github.com/indico/indico/) (which relies on pycountry to provide County lists) with an international audience.
Thanks

**Comment by atocharnaud** [comment]
pycountry is used at the OECD by multiple analysts. Getting this chance would be very welcomed.

**Comment by arcticlinux** [comment]
Not completely related but I was trying to look up countries based on their currency, which seems to work most of the time, but doesn't work for the Turkish Lira, by doing:
```
currency_code = 'TRY
pycountry_currency = pycountry.currencies.get(alpha_3=currency_code)
    if pycountry_currency:
        country = pycountry.countries.get(numeric=pycountry_currency.numeric)
```

**Comment by justmeben** [comment]
Any update?

**Comment by embray** [comment]
This was also updated in ISO-3166: https://www.iso.org/obp/ui/#iso:code:3166:TR

**Comment by alejandrofm** [comment]
Any news?

**Comment by alejandrofm** [comment]
Any news about this? I'm using Pycountry + IF 'Türkiye' THEN... there are several PRs that could make this a better lib.

**Comment by zware** [comment]
This has been fixed in #165, which updated us to iso-codes v4.15.0.
